src/sentry/static/sentry/app/views/issueList/overview.tsx

```diff
@@ -193,14 +191,6 @@ class IssueListOverview extends React.Component<Props, State> {
       this.fetchTags();
     }
 
-    // Wait for saved searches to load before we attempt to fetch stream data
-    if (this.props.savedSearchLoading) {
-      return;
-    } else if (prevProps.savedSearchLoading) {
-      this.fetchData();
```

Comment:
If you're using a saved query does the page do two issue search requests? One for the initial state and then another for the selected search?

Reply:
Ah yeah you're right, its about the same then except now it cancels the request

Reply:
Maybe we can pass a query parameter to `/issues/` to use the pinned query if it exists so we don't have to wait for it to come back. Saved search could do something similar